### PR TITLE
[Travis] Add PHP 7

### PR DESCRIPTION
**Pull request by egeloen**
This PR adds PHP 7 to the travis build matrix.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ php:
     - 5.4
     - 5.5
     - 5.6
+    - nightly
```
Comment (GrahamCampbell) [inline .travis.yml]: should be 7.0 really

Comment (GrahamCampbell) [inline .travis.yml]: nightly makes no sense, beause in one year, nightly will not mean 7.0, and then 7.0 will be untested

Comment (egeloen) [inline .travis.yml]: Thanks! Fixed

**Comment by egeloen** [comment]
@GrahamCampbell Do you know why PHP tries to load a CakePHP class (which is a reserved keyword in PHP7) whereas I have disabled the tests/loading of CakePHP when we are on PHP7?

**Comment by GrahamCampbell** [comment]
Because you put that skip in the wrong place.

**Comment by GrahamCampbell** [comment]
It can't go in a static method that's only called once for the whole class. It needs to go in a method called before each test.

**Comment by egeloen** [comment]
Okay, thanks! I though we can skip a test in a set up before class...

**Comment by egeloen** [comment]
Shit, shit, shit, It does not matter :s I will need to compile PHP7 and see what's going on with CakePHP...

**Comment by egeloen** [comment]
This PR is in standby since #103 is merged.

**Pull request merged**